src/App.tsx

```diff
@@ -34,6 +35,21 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods.sort((a, b) => a.localeCompare(b));
+      break;
+    case SortType.LENGTH:
+      visibleGoods.sort((a, b) => a.length - b.length);
```

Comment:
DRY, use sort just once and use switch case in callback

Reply:
@ArturErmolenko  Is that good to use `switch case` many times inside of an iterator instead of one time and then start the iterator?

Reply:
you use switch just once & passing right callback into sort method

Reply:
Oh, got it, thank you 🙂